assets/kube-apiserver/config.yaml

```diff
@@ -22,7 +22,7 @@ admission:
         kind: PodSecurityConfiguration
         apiVersion: pod-security.admission.config.k8s.io/v1beta1
         defaults:
-          enforce: "privileged"
+          enforce: "restricted"
```

Comment:
Please also move the API version to `pod-security.admission.config.k8s.io/v1`.

Reply:
Thanks... change made